app/templates/app/styles/pages/_styleguide.scss

```diff
@@ -0,0 +1,113 @@
+/* ==========================================================================
+   WEBSITE STYLE GUIDE
+   ========================================================================== */
+
+// Global
+.styleguide {
+	#main {
```

Comment:
I highly recommend avoid applying styles to `id`'s. Though tempting sometimes, the specificity issues they can cause down the road can be painful.